packages/auth/graphql/schema-types.js

```diff
@@ -39,6 +39,11 @@ type User {
   preferredFirstFactor: SignInTokenType
   enabledSecondFactors: [SignInTokenType!]!
   eventLog: [EventLog!]!
+  # is this the user of the requesting session
+  isMe: Boolean
```

Comment:
after sleeping about it I'd still prefer `isCurrentSession` or `isInCurrentSession`. Because if one provides an auth token, and they are not logged in, the requester sort of is me, just not authenticated – just not in the session.

Reply:
I see, the case with the `accessToken` is tricky, but `User.isCurrentSession` is not self explaning either. Why should a user be a session?
How about `User.isOwnerOfCurrentSession` or `isUserOfCurrentSession`?

Reply:
`isUserOfCurrentSession` :)

Reply:
or `User.isOfCurrentSession` or `User.isInCurrentSession` or even shorter `User.isCurrentSession` :P